cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -263,6 +264,14 @@ func Validate(args *Args, cfg *config.Config) error {
 		return errors.New("invalid empty root directory")
 	}
 
+	if args.RootDir == defaultRootDir {
+		if entries, err := os.ReadDir(oldDefaultRootDir); err == nil {
```

Comment:
Old users will confuse if the error `default root directory is changed to /var/lib/containerd-nydus` be thrown and snapshotter exits, just give a warning for that?

Reply:
Changed to a warning message